tools/make_version_file.py

```
@@ -8,7 +8,7 @@
     strfile = fvers.read()
     fvers.close()
 except:
-    None
+    pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)